packages/nimble-components/src/banner/specs/README.md

```diff
@@ -0,0 +1,167 @@
+# Banner
+
+## Overview
+
+The banner is a component used to display a persistent notification to a user.
+
+### Background
+
+[Design specification](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/29c405f7-08ea-48b6-973f-546970b9dbab/)
+
+[GitHub issue](https://github.com/ni/nimble/issues/305)
+
+### Non-goals
+
+-   Arbitrary HTML content
+-   A container component that would enforce design constraints on multiple banners displayed together
```

Comment:
I don't remember from Brandon's spacing tokens demo whether he had a token for a 1-px spacing, but we need one. Though I still don't understand the point of tokens if they are just something like `--ni-nimble-spacing-1px`, since it wouldn't make sense to change what the value associated with that is. We'll definitely have some documentation about how to space them apart when stacking them.

Reply:
We haven't committed to a change in token naming yet and having an SCSS defined token value will help clients for when we have a change.

Going on our [current name formatting](https://github.com/ni/nimble/blob/main/packages/nimble-components/CONTRIBUTING.md#token-naming) I'd say a token like `banner-gap-size` (element: banner, part: gap, state: none/default, type: size), but open to other ideas following our convention. I think a token would be more useful in the interim than citing a 1px gap in the documentation until we commit to a change in size token architecture.